PalasoUIWindowsForms/ImageToolbox/Cropping/ImageCropper.cs

```diff
@@ -358,33 +359,26 @@ public Image GetCroppedImage()
 
 			try
 			{
-				double z = 1.0/GetImageToCanvasScaleFactor();
-
-
-				int width = Math.Max(1, _rightGrip.Value - _leftGrip.Value);
-				int height = Math.Max(1, _bottomGrip.Value - _topGrip.Value);
-				var selection = new Rectangle((int) Math.Round(z*_leftGrip.Value),
-											  (int) Math.Round(z*_topGrip.Value),
-											  (int) Math.Round(z*width),
-											  (int) Math.Round(z*height));
-
 				//jpeg = b96b3c  *AE* -0728-11d3-9d7b-0000f81ef32e
 				//bitmap = b96b3c  *AA* -0728-11d3-9d7b-0000f81ef32e
 
-				Bitmap bmp = _originalImage as Bitmap;
-				if (bmp == null)
-					throw new ArgumentException("No valid bitmap");
-
-				CheckForInvalidImage();
-
 				//NB: this worked for tiff and png, but would crash with Out Of Memory for jpegs.
 				//This may be because I closed the stream? THe doc says you have to keep that stream open.
 				//Also, note that this method, too, lost our jpeg encoding:
 				//          return bmp.Clone(selection, _image.PixelFormat);
 				//So now, I first copy it, then clone with the bounds of our crop:
 
-				using (var copy = new Bitmap(_originalImage)) //**** here we lose the jpeg rawimageformat, if it's a jpeg. Grrr.
+				using (var copy = new Bitmap(_savedOriginalImage.Path)) //**** here we lose the jpeg rawimageformat, if it's a jpeg. Grrr.
```

Comment:
I couldn't determine if one method is better than the others. It looks like they all call GDI+ to do the actual copying.